Step 1 (#4)
aws-samples/aws-waf-workshop: docs/step-1.md

* XSS: `<your-endpoint>/?<SCRIPT>alert(“Cookie”+document.cookie)</SCRIPT>`

## Part III - HTTP Flood (AWS Lambda log parser)

To test HTTP Flood, you can simulate an WAF log file deliver event. For that:

* Go to the CloudFormation stack's `Outputs` tab and search for the value defined for `WafLogBucket`
* Go to S3, and upload [this file](files/waf-access-log-sample.gz) to the `WafLogBucket` bucket
* Wait a few seconds (while the log parser function processes the new WAF log file)
* Check if the file `<stack_name>-waf_log_out.json` was added to the `WafLogBucket` bucket
* Go to [AWS WAF console](https://console.aws.amazon.com/wafv2/home?#/webacls) and check if `HTTP Flood` rule contains any IP listed

## Part IV - Scanners & Probe (Amazon Athena log parser)

To test Scanners & Probe, you can simulate a CloudWatch event running a query in a bucket that contains a sample access log file.
For that:
* Go to the S3 bucket that you set as Access Log Bucket. If you don't remember, go to stack's `Outputs` tab and search for the value defined for `AppAccessLogBucket`
* Create a new folder and name it AWSLogs and upload [this file](files/alb-access-log-sample.gz) to it

> To not have to wait until this file be process during the regular *Scanners & Probe* scan, you can run an on-demand *Scanners & Probe* scan. For that:

Reviewer: I would suggest this be rewritten as
```suggestion
This file will be processed during the next scheduled *Scanner and Probe* scan. Rather than wait, we will run it on demand.
```
